networkit/cpp/flow/test/EdmondsKarpGTest.cpp

```diff
@@ -38,6 +38,9 @@ TEST_F(EdmondsKarpGTest, testEdmondsKarpP1) {
     EXPECT_EQ(1, edKa.getFlow(5, 6));
 
     std::vector<node> sourceSet(edKa.getSourceSet());
+    for (auto u : sourceSet) {
+        std::cout << "u: " << u << std::endl;
+    }
```

Comment:
This was used for debugging right? If you don't need it anymore it should be removed.